WORKSPACE

```diff
@@ -79,6 +76,23 @@ paket2bazel_dependencies()
 load("//csharp/nuget:paket.csharp_deps.bzl", csharp_deps = "csharp_deps")
 csharp_deps()
 
+# Load //builder/go
+load("@vaticle_dependencies//builder/go:deps.bzl", go_deps = "deps")
+go_deps()
+load("@io_bazel_rules_go//go:deps.bzl", "go_rules_dependencies")
+go_rules_dependencies()
+
+load("//go:go_versions.bzl", "register_all_toolchains")
+register_all_toolchains()
+
+# gazelle:repo bazel_gazelle <- Used to tell gazelle that it is loaded in a macro.
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
+load("//go:deps.bzl", "go_repositories")
+# gazelle:repository_macro go/deps.bzl%go_repositories
+go_repositories()
+gazelle_dependencies()
+
+
```

Comment:
I don't want to be noisy, but let's just aim to notice such things by yourself, it is a good tone =)